src/elements/x-safe.js

```diff
@@ -160,6 +160,11 @@ export default class XSafe extends MixinRedux(XElement) {
     }
 
     _onPropertiesChanged(changes) {
+        if (changes.activeWallet === null) {
+            // user logged out of all wallets
+            accountManager.onboard();
```

Comment:
I guess the redirect happens top-level, and nothing serious happens currently in this function after this block, but a `return;` would probably not hurt ;)

Reply:
ok :D